src/chi/javascript/core/util.js

```diff
@@ -427,6 +427,20 @@ export class Util {
     }
   }
 
+  static calculateHiddenElementHeight(elem) {
+    let elemHeight;
+
+    elem.style.position = 'absolute';
```

Comment:
You should save the previous values of this style properties so you can roll-back to them. It could have these attributes already set to other values. 
Some contents, depending on the width of the container could larger or smaller height. How do you control this?